package-lock.json

```diff
@@ -6936,8 +6936,7 @@
           "version": "3.0.0",
           "resolved": "https://registry.npmjs.org/get-stream/-/get-stream-3.0.0.tgz",
           "integrity": "sha1-jpQ9E1jcN1VQVOy+LtsFqhdO3hQ=",
-          "dev": true,
-          "optional": true
+          "dev": true
```

Comment:
Can we revert these `package-lock.json` changes? They aren't relevant to this PR.